server/node.go

```diff
@@ -116,7 +117,7 @@ func BootstrapCluster(clusterID string, eng engine.Engine) (*client.KV, error) {
 	// Create a KV DB with a local sender.
 	lSender := kv.NewLocalSender()
 	localDB := client.NewKV(kv.NewTxnCoordSender(lSender, clock), nil)
-	s := storage.NewStore(clock, eng, localDB, nil)
+	s := storage.NewStore(clock, eng, localDB, nil, multiraft.NewLocalRPCTransport())
```

Comment:
Once we have the real transport impl we should move to that right? Do we ever need to use the local transport?
Just wondering if we need a factory or configuration to drive this.